### DISCUSSION TITLE
Disable Text on the Checkbox

### DISCUSSION THREAD

**Discussion by BetrickBZD** [TomSchimansky/CustomTkinter#1361] [Q&A]
Hi I need help to disable the text parameter on the Checkbox when i dont write anything it looks like this
![grafik](https://user-images.githubusercontent.com/128264757/226145471-3cd96b57-7e07-441b-85d7-cb21435f9e63.png)
I dont whant any text how can i do it ?

```
checkbox = customtkinter.CTkCheckBox(master=frame,onvalue="on", offvalue="off")
checkbox.grid(pady=(10,0),padx=(0,0),row=1,column=1)
```

**Comment by ElectricCandlelight** [comment]
customtkinter.CTkCheckBox(master=frame, text="",onvalue="on",  offvalue="off")

**Reply by BetrickBZD** [reply]
Hi thanks for the solution, but the problem with this method is that the label part is still there just invisble becouse of that the checkbox does not center it self to the frame do someone know a solution for it?
![grafik](https://user-images.githubusercontent.com/128264757/226212192-a610da2d-154e-4c0e-8bbb-604f9a1f46b7.png)

**Reply by avalon60** [reply]
Have you tried:
ctk_checkbox.configure(text=''')

**Comment by brodzina** [comment]
try set width attribute default is 100
`checkbox = customtkinter.CTkCheckBox(master=frame,onvalue="on", offvalue="off", width=30)`

**Reply by bfedie518** [reply]
The default width of the checkbox itself is 24, so that's probably what you want.

**Comment by bfedie518** [comment]
Related to this, I just made a pull request #1465 that allows for choosing on which side of the checkbox the text appears.
